content/en/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch.md

```diff
@@ -304,7 +304,7 @@ spec:
 Patch your Deployment:
 
 ```shell
-kubectl patch deployment retainkeys-demo --type merge --patch-file patch-file-no-retainkeys.yaml
+kubectl patch deployment retainkeys-demo --patch-file patch-file-no-retainkeys.yaml
```

Comment:
```suggestion
kubectl patch deployment retainkeys-demo --type strategic --patch-file patch-file-no-retainkeys.yaml
```

Although `strategic` is the default, making it explicit helps teach readers.

Reply:
Very useful suggestion, Thanks